modules/op-disabling-tekton-hub-authorization-after-upgrade.adoc

```diff
@@ -0,0 +1,91 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-hub-with-openshift-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="disabling-tekton-hub-authorization-after-upgrade.adoc_{context}"]
+= Disabling {tekton-hub} authorization after upgrading the {pipelines-title} Operator from 1.7 to 1.8
+
+[role="_abstract"]
+When you install {tekton-hub} with {pipelines-title} Operator 1.8, the login authorization and ratings for the Hub artifacts are disabled for the default installation. However, when you upgrade the Operator from 1.7 to 1.8, the instance of the Hub on your cluster does not automatically disable the login authorization and ratings. 
+
+To disable login authorization and ratings for {tekton-hub} after upgrading the Operator from 1.7 to 1.8, perform the steps in the following procedure.
+
+[discrete]
+.Prerequisites
+* Ensure that the {pipelines-title} Operator is installed in the default `openshift-pipelines` namespace on the cluster.
+
+[discrete]
+.Procedure 
+
+Starting with {pipelines-title} 1.8, you can automatically install and deploy {tekton-hub} with default configurations on a cluster by performing the following steps:
+
+. Delete the existing {tekton-hub} API secret that you created while manually installing {tekton-hub} for Operator 1.7.
++
+[source,terminal]
+----
+$ oc delete secret tekton-hub-api -n <targetNamespace> <1>
+----
+<1> The common namespace for the {tekton-hub} API secret and the {tekton-hub} CR. By default, the target namespace is `openshift-pipelines`.
+
+. Delete the `TektonInstallerSet` object for the {tekton-hub} API.
++
+[source,terminal]
+----
+$ oc delete tektoninstallerset tekton-hub-api-mv5dv
+----
++
+[NOTE]
+====
+After deletion, the Operator automatically creates a new {tekton-hub} API installer set.
+====
++
+Wait and check the status of the {tekton-hub}. Proceed to the next steps when the `READY` column displays `True`.
++
+[source,terminal]
+----
+$ oc get tektonhub hub
+----
++
+.Sample output
+[source,terminal]
+----
+NAME   VERSION        READY   REASON   APIURL                                                                                                  UIURL
+hub    1.8.0          True             https://tekton-hub-api-openshift-pipelines.apps.ci-ln-279pdpk-72292.origin-ci-int-gce.dev.rhcloud.com   https://tekton-hub-ui-openshift-pipelines.apps.ci-ln-279pdpk-72292.origin-ci-int-gce.dev.rhcloud.com
+
+----
+
+. Delete the `ConfigMap` object for the {tekton-hub} UI.
++
+[source,terminal]
+----
+$ oc delete configmap tekton-hub-ui -n <targetNamespace> <1>
+----
+<1> The common namespace for the {tekton-hub} UI and the {tekton-hub} CR. By default, the target namespace is `openshift-pipelines`.
+
+. Delete the `TektonInstallerSet` object for the {tekton-hub} UI.
++
+[source,terminal]
+----
+$ oc delete tektoninstallerset tekton-hub-ui-c8v7s
```

Comment:
Suffix `c8v7s` is auto-generated and is unique for given deployment. We show users how to get that (`oc get tektoninstallerset`) or provide a command that can be copied & pasted such as
```
oc get tektoninstallerset -o name | grep tekton-hub-ui | xargs oc delete
```